### DISCUSSION TITLE
How to Update

### DISCUSSION THREAD

**Discussion by airbornetrooper82573** [mrlt8/docker-wyze-bridge#196] [Q&A]
This seems to be updated fairly often.... which is great! But how do we update? Can I get an "explain like I'm 5" version?

**Comment by mrlt8** [comment]
Something like this in the same directory as your docker-compose.yml should work:
```sh
docker-compose pull # Pull new image
docker-compose up -d # Restart container in detached mode
docker image prune # Remove old images
```

I believe [watchtower](https://containrrr.dev/watchtower/) is a popular way to automate updates, but I don't use it myself, so can't comment much on it.
